packages/material-ui/src/BusyButton/BusyButton.js

```diff
@@ -0,0 +1,144 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import clsx from 'clsx';
+import { chainPropTypes } from '@material-ui/utils';
+import withStyles from '../styles/withStyles';
+import capitalize from '../utils/capitalize';
+import Button from '../Button';
+import CircularProgress from '../CircularProgress';
+
+export const styles = () => ({
+  /* Styles applied to the root element. */
+  root: {},
+  /* Styles applied to the root element if `pending={true}`. */
+  pending: {},
+  /* Styles applied to the pendingIndicator element. */
+  pendingIndicator: {
+    position: 'absolute',
+    visibility: 'visible',
+    display: 'flex',
+  },
+  /* Styles applied to the pendingIndicator element if `pendingPosition="center"`. */
+  pendingIndicatorCenter: {
+    left: '50%',
+    transform: 'translate(-50%)',
+  },
+  /* Styles applied to the pendingIndicator element if `pendingPosition="start"`. */
+  pendingIndicatorStart: {
+    left: 16,
```

Comment:
It feels we can better align the icon

**Before**
<img width="278" alt="before" src="https://user-images.githubusercontent.com/3165635/84565641-b8fdc980-ad6a-11ea-8810-fad59edcd4c8.png">

**After**
<img width="261" alt="after" src="https://user-images.githubusercontent.com/3165635/84565640-b8653300-ad6a-11ea-8ffa-0ada099a38a3.png">

```suggestion
    left: 14,
```